### ISSUE TITLE
Updating the FriendsOfSymfony logo

### ISSUE THREAD

**Issue by jmikola** [FriendsOfSymfony/friendsofsymfony.github.io#23]
Provided this is kosher with Symfony2's [trademark rules](http://symfony.com/trademark)... any feedback on updating the FOS logo?

![FOSome.jpg](http://i.imgur.com/5VF3t.png)

Or perhaps an inverted, lighter variation of that? I'm only being half-serious here, but I do think the existing FOS avatar could use a refresh.

**Comment by lsmith77** [comment]
+1 on anything with a smilie .. cant the `sf` be the nose?

**Comment by willdurand** [comment]
Will be quite hard for @Stof to review this logo :D

btw +1

**Comment by stof** [comment]
I agree that the current avatar can be improved. So +1 for the search of a new logo

However, I'm not in love with this proposal (don't ask me to explain why, maybe because it feels too much like the Sf logo at first look, especially when looking at my screen in certain positions making the color darker :) ).

If anybody else has idea, feel free to post them here too.

**Comment by ornicar** [comment]
I'm all for a new logo, and I salute the vaillant effort but am looking forward more propositions.

I personally am a terrible designer.

**Comment by jmikola** [comment]
@lsmith77 Any logo has to display well when sized as an icon, so an "sf" as the nose would quickly become illegible when the image is shrunk.

No hard feelings about the negative feedback. This was intended to be a silly suggestion to get the conversation started. I'll see what else I can whip up in my spare time going into the weekend. Anyone else is welcome to contribute as well :)

**Comment by merk** [comment]
+0, dont mind either way :)

**Comment by stof** [comment]
anybody with other ideas ?

**Comment by jmikola** [comment]
What about Fozzie Bear doing an @lsmith77 pose?

![](http://i.imgur.com/zoF8sl.jpg)

**Comment by inelgnu** [comment]
<img src="https://p.twimg.com/AlrkAlwCEAE2_Mz.png">

**Comment by jcrombez** [comment]
Here is my random try :

<img src="http://files.easytek.fr/fos-logo-0.1.png" />

or a flat version :

<img src="http://files.easytek.fr/fos-logo-0.1-flat.png" />

**Comment by stof** [comment]
@inalgnu your proposal is fun but I fear a trademark issue...

**Comment by willdurand** [comment]
I like the @inalgnu's proposal

**Comment by mmoreram** [comment]
I'll think about it.. XD Is important to have a good logo XD

**Comment by lsmith77** [comment]
as for the trademark, i guess its not allowed .. tough we could always try to get an exception ..

**Comment by ornicar** [comment]
I don't have a photo of my face right now, but I have one of my ass. There is no trademark on it!

**Comment by jcrombez** [comment]
@ornicar The comment you replied to being now removed, your message is kind of...awkward :D

**Comment by cordoval** [comment]
yes it was my fault, i removed them, sorry apologize to all

**Comment by ornicar** [comment]
what?? why the hell was the previous comment removed? it was okay

bah nevermind, I'll leave mine anyway. Deal with it :D

**Comment by jcrombez** [comment]
<img src="http://files.easytek.fr/fos-logo-0.2w.png" />

<img src="http://files.easytek.fr/fos-logo-0.3w.png" />

**Comment by rleal** [comment]
Another smiler version :)

![FOS Logo](http://img836.imageshack.us/img836/5505/foslogo.png)

**Comment by datiecher** [comment]
So @ornicar is (literally) showing his ass on Symfony land again! =D

**Comment by jmikola** [comment]
Does anyone want to take a shot a deconstructing the Symfony logo into F and S for FOS? I'm curious what that might look like.

**Comment by jcrombez** [comment]
@jmikola you mean like that ?

<img src="http://files.easytek.fr/sf2-copy-fs.png" />

or like that ?

<img src="http://files.easytek.fr/sf2-copy-fos.png" />

I guess it may be a problem in term of trademark...

**Comment by merk** [comment]
i like that direction, but we'd probably want to seek permission before using something like that

**Comment by jmikola** [comment]
Certainly, if we do anything with the official Symfony logo, we should run it by @fabpot to ensure it's consistent with the [trademark policy](http://symfony.com/trademark). I imagine that deconstructing the logo might not be kosher (but I was more or less curious what the results would look like :)

@jcrombez's "fos" logo looks quite nice. And of all the non-Symfony logo submissions, I like @rleal's the most. I think it would also scale down to GitHub avatar size quite well, which is important.

**Comment by adrienbrault** [comment]
Hey everyone

![z](http://f.cl.ly/items/2Y1n1B271f1K0t1u403w/FOS%20Logo.png)

https://docs.google.com/drawings/d/1s5WzA33dAZMzQDRlz3qjvbEGVcNGpxMJo4CuH-53WrY/edit
